livre/core/pipeline/Executable.h

```diff
@@ -17,28 +17,28 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#include <livre/core/pipeline/Processor.h>
+#ifndef _Executable_h_
+#define _Executable_h_
+
+#include <livre/core/types.h>
 
 namespace livre
 {
 
-Processor::Processor()
+/**
+ * The Executable class is the base clas for objects that can be executed by workers.
```

Comment:
typo clas

Style: In doxygen I already see that this is the Executable class. I prefer:

'Base class for objects that can be executed by workers' , i.e., DRY.